<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winReport</class>
 <widget class="QDialog" name="winReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>148</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>148</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Построить отчет</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Диапазон отчетности</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>с</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbFrom">
        <property name="suffix">
         <string> г</string>
        </property>
        <property name="minimum">
         <number>2000</number>
        </property>
        <property name="maximum">
         <number>3000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>по</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbTo">
        <property name="suffix">
         <string> г</string>
        </property>
        <property name="minimum">
         <number>2000</number>
        </property>
        <property name="maximum">
         <number>3000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbMake">
       <property name="text">
        <string>Построить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="pbrStatus">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>%v/%m</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
